src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,67 @@
+package racingcar;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Random;
+import org.junit.jupiter.api.Test;
+
+public class CarTest {
+
+  @Test
+  void getName() {
+    Car car = new Car("test");
+    assertEquals("test", car.getName());
```

Comment:
값을 꺼내서 비교하기보다는,
```suggestion
    assertThat(car).isEqualTo(new Car("test"));
```
객체간 비교하면 어떨까요?
객체간 비교를 하려면 무엇을 구현해야할까요?